qt/python/mantidqt/widgets/algorithmprogress/dialog_widget.py

```diff
@@ -38,8 +39,10 @@ class AlgorithmMonitorDialog(QDialog):
     """
     Displays progress of all running algorithms.
     """
+
     def __init__(self, parent, model):
         super(AlgorithmMonitorDialog, self).__init__(parent)
+        self.parent = parent
```

Comment:
There is no need to save `parent` as it is passed to the base class constructor and can be accessed via `self.parent()` method.